ovos_utils/file_utils.py

```diff
@@ -0,0 +1,240 @@
+import collections
+import csv
+import re
+import os
+from os import walk
+from os.path import splitext, join, dirname
+
+from ovos_utils.bracket_expansion import expand_options
+from ovos_utils.log import LOG
+
+
+def resolve_ovos_resource_file(res_name):
```

Comment:
Anything new in this file, or just moved funcs?

Reply:
new vocab loading funcs below